userguide/snippets/es24m_rails.rst

```diff
@@ -52,10 +52,15 @@ click in place. Repeat this process for the second rear rail.
 Mount the Unit in the Rack
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-**Caution: Two people are required to safely lift the chassis for rack
-installation or removal.** Do not install drives until after the
-chassis has been installed in the rack, and remove all drives before
-removing the chassis from the rack.
+.. caution:: The unit weighs about 40 pounds.
```

Comment:
Spec sheet for ES24 says 40 lbs unloaded, 76 lbs when fully loaded. At minimum, I'd remove "about" -> "The unit weighs 40 pounds." Might want to rework a bit to show the loaded weight too: "The unit weighs 40 pounds empty and 76 pounds when fully loaded with drives."

Reply:
Reviewed M-Series BSG and noticed it uses this file for the rack mount instructions too. This caution needs to show weights for both the ES24 and the M-Series.

Reply:
There is some margin for error and changes due to packaging.  The "about" is good to show that some variation is expected and the quoted weights should be verified before someone ships one.